Xamarin.PropertyEditing.Mac/Controls/BaseNumericEditorControl.cs

```diff
@@ -72,7 +72,7 @@ protected override void SetEnabled ()
 		protected override void UpdateAccessibilityValues ()
 		{
 			NumericEditor.AccessibilityEnabled = NumericEditor.Enabled;
-			NumericEditor.AccessibilityTitle = ViewModel.Property.Name + " Numeric Editor"; // TODO Localization
+			NumericEditor.AccessibilityTitle = string.Format (" {0} {1}", ViewModel.Property.Name, LocalizationResources.AccessibilityNumeric);
```

Comment:
The localizable string should be `{0} Numeric Editor`and we should be calling the api like: `Xamarin.PropertyEditing.Mac.Strings.AccessibilityNumeric (ViewModel.Property.Name)`. If we reference this nuget ( https://www.nuget.org/packages/netfx-System.StringResources/ ) and set the resx files up like we did in all of the csproj files in the designer repository it will autogenerate the right c# API for you.

We then need to add a comment explaining what `{0}` is for and what an example usage of it would look like. For example `Corner Radius Numeric Editor`.

Reply:
Ah ok, thanks for the clarification.